config/eurotronic/eur_spiritz.xml

```diff
@@ -1,83 +1,110 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--
+Eurotronic Spirit Z-Wave Plus Thermostat
+Product website: https://www.eurotronic.org/produkte/spirit-z-wave-plus.html
+Tech manual: https://www.eurotronic.org/fileadmin/user_upload/eurotronic.org/Produktbilder/spirit_z_wave_plus/Spirit_Z-Wave_BAL_web_EN_view_04.pdf
+-->
 <Product xmlns='http://code.google.com/p/open-zwave/'>
-    <CommandClass id="67" base="0" typeInterpretation="A" />
-    <CommandClass id="64" >
+    <!-- Multilevel switch for setting valve opening -->
+    <CommandClass id="38">
+        <Value type="byte" genre="user" index="0" label="Valve Opening" units="%" min="0" max="100" value="0" />
+    </CommandClass>
+
+    <!-- Thermostat mode -->
+    <CommandClass id="64">
         <Instance index="1" />
         <Value type="list" genre="user" instance="1" index="0" label="Mode">
-                <Item label="Off" value="0" />
-                <Item label="Heat" value="1" />
-                <Item label="Heat Econ" value="11" />
-                <Item label="Full Power" value="15" />
-                <Item label="Manufacturer Specific" value="31" />
+            <Help>
+                Off: No heating, only frost protection.
+                Auto: Room temperature will be kept at the configured setpoint.
+                Eco: Energy save heating mode. Room temperature will be lowered to the configured eco setpoint in order to save energy.
+                Full Power: Full power heating. This mode is left automatically after 5 minutes.
+                Manufacturer Specific: Direct valve control mode. The valve opening percentage can be controlled using the switch multilevel command class.
+            </Help>
+            <Item label="Off" value="0" />
+            <Item label="Auto" value="1" />
+            <Item label="Eco" value="11" />
+            <Item label="Full Power" value="15" />
+            <Item label="Manufacturer Specific" value="31" />
         </Value>
         <SupportedModes>
-                <Mode index="0" label="Off" />
-                <Mode index="1" label="Heat" />
-                <Mode index="11" label="Heat Econ" />
-                <Mode index="15" label="Full Power" />
-                <Mode index="31" label="Manufacturer Specific" />
+            <Mode index="0" label="Off" />
+            <Mode index="1" label="Auto" />
```

Comment:
This is called "Heat" in the Zwave specification. 0x03 would be Auto.
Also  "Eco" / "Heat Eco" will get called "Heat Eco" when discovered dynamically, so also leave it as is (Only added these  values for the Manufacturer Specific to show up as it is not )

Reply:
You're right. I tried to be a bit more accurate with the wording, as it won't "heat" if you put it into "heat" mode. It will only heat if necessary to reach the setpoint temperature.

But I agree, following conventions is more important than being accurate when it comes to naming.

But let's keep naming it "Heat Eco" as opposed to "Heat Econ", which sounds more like an "electronic convention" rather than "economical" 😁